pds_doi_service/core/util/config_parser.py

```diff
@@ -18,6 +18,16 @@
 
 class DOIConfigUtil:
 
+    @staticmethod
+    def _resolve_relative_path(parser):
+        # resolve relative path with sys.prefix base path
+        for section in parser.sections():
+            for (key, val) in parser.items(section):
+                if key.endswith('_file') or key.endswith('_dir'):
+                    parser[section][key] = os.path.abspath(os.path.join(sys.prefix, val))
+
```

Comment:
A nice to have would be logging the before/after of the path resolution, perhaps only at the debug level